### PR TITLE
83 - Trigger for Firebase (LinkedIn Post saved)

### PR DESCRIPTION
**Pull request by mohanhasti**
JIRA Ticket link: [PC-83](https://makeitmvp.atlassian.net/browse/PC-83)

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring

## Description

- Changes made?
added Firebase connectivity to post the content when the LinkedIn post button is pressed.

- Reason for changes?
to add the connectivity to Firebase for linkednAPI

- Behavior before?
- Behavior after?
- How did you test?

## Checklist

- [ ] I have performed a self-review of my code.
- [x] I have manually tested the new feature and found it to be working as expected.
- [] All commits in this PR are related to the issue.
- [x] Base branch for this PR is set to `develop`.
- [x] JIRA ticket # and my initials are noted in this PR's name (format example: #57-EB Add Pull Request template)

**Reminder**: every PR needs at least 1 peer approval before merging to the `develop` branch. Once a main ticket has been completed, along with all subtasks, associated code can be merged to `main`.

[PC-83]: https://makeitmvp.atlassian.net/browse/PC-83?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mohanhasti** [comment]
JIRA Ticket link: [PC-80](https://makeitmvp.atlassian.net/browse/PC-80)

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring

## Description

- Changes made?
added feature to pass the content onClick of a button.

- Reason for changes?
so that the content can be sent to the LinkedIn API to be posted
- Behavior before?
- Behavior after?
- How did you test?
Browser inspect tools

## Checklist

- [ ] I have performed a self-review of my code.
- [x] I have manually tested the new feature and found it to be working as expected.
- [x] All commits in this PR are related to the issue.
- [x] Base branch for this PR is set to `develop`.
- [x] JIRA ticket # and my initials are noted in this PR's name (format example: #57-EB Add Pull Request template)

**Reminder**: every PR needs at least 1 peer approval before merging to the `develop` branch. Once a main ticket has been completed, along with all subtasks, associated code can be merged to `main`.


[PC-80]: https://makeitmvp.atlassian.net/browse/PC-80?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by eburdekin** [comment]
@mohanhasti looks like this PR also includes the commit for the LinkedIn API auth code. Thought you were going to merge that update into 'develop' first?

**Comment by mohanhasti** [comment]
Yes, the code has both the features(firebase and auth). But I think we can just merge it into develop. It would work.

**Comment by eburdekin** [comment]
@mohanhasti I still need to review the added Firebase code, but this isn't following our team's process. Separate features should be handled in separate branches/PRs - that way they map to the Jira ticket numbers in our template

**Comment by IguanasEverywhere** [comment]
Hey @mohanhasti , I'm getting an error from from firebase when I load up the home component (see screenshot). I've made sure to run `npm install` and double-check the `.env` file for the keys. Is there something else I'm missing re: the setup here? I'm also getting an error about a missing client_id when I went to look at the "/linkedin" route.
firebaseError
linkedinError

**Comment by mohanhasti** [comment]
Ohh I think you might be using the" Content" firebase. We are no longer using it. So I think you have to update the firebase API keys to the communiti firebase. 
For /LinkedIn, we have to put in client id and secret in the .env.

**Comment by mohanhasti** [comment]
> @mohanhasti I still need to review the added Firebase code, but this isn't following our team's process. Separate features should be handled in separate branches/PRs - that way they map to the Jira ticket numbers in our template

@eburdekin do I just create a new branch and PR for LinkedIn again ?

**Comment by IguanasEverywhere** [comment]
> Ohh I think you might be using the" Content" firebase. We are no longer using it. So I think you have to update the firebase API keys to the communiti firebase. For /LinkedIn, we have to put in client id and secret in the .env.

Ah, you're right, my mistake. Have you shared with us the CLIENT_ID for the LinkedIn? I saw the link you gave in slack to the account itself, but I know I'll need the credentials to test on my end.

**Comment by eburdekin** [comment]
@mohanhasti it's fine for this one but if you can do that moving forward, it'll help keep things organized and get features merged into `develop` as they're approved, which will help us all keep the most updated code on our machines.

**Comment by mohanhasti** [comment]
Got it, I will go ahead and merge it. Thanks.

**Comment by eburdekin** [comment]
@mohanhasti I don't think we're ready to merge with the Firebase updates yet. There are also going to be merge conflicts with Scott's updates

File: src/main.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 import "./styles/partials/_global.scss";
 
 const Home = () => {
   return (
     <div className="home">
       <h1 className="home__title"> Welcome to COMMIT AI!</h1>
       <Link to="/prompt">Prompt the AI here</Link>
-      <UserList />
+      {/* <UserList /> */}
+      <MessageList />
```
Comment (eburdekin) [inline src/main.js:12]: Confirming I'm seeing the MessageList here from the Firebase db 👍

**Pull request merged**